test/intl402/NumberFormat/prototype/format/format-significant-digits-precision.js

```diff
@@ -23,8 +23,8 @@ var numberingSystems = [
 ];
 var testData = {
     // Ref tc39/ecma402#128
```

Comment:
this comment links to the still-open issue: https://github.com/tc39/ecma402/issues/128 - are we confident about the conclusion both there and here?

Reply:
are you sure I added this ref? I can't find a git log doing it. I'm not challenging you, but actually I don't remember the reason it's added here or anything. In any case, I can review this for sure.

Reply:
https://github.com/tc39/test262/commit/27b156c31e644166e606df21476e39b9a75f4fbc

found it. That was a just a request.